README.adoc

```diff
@@ -0,0 +1,75 @@
+[[minishift-addons]]
+= Minishift Add-ons
+:icons:
+:toc: macro
+:toc-title:
+:toclevels: 1
+
+toc::[]
+
+[[welcome-to-minishift]]
+== Welcome to the Minishift add-ons repository!
```

Comment:
@thatdocslady This is a long sentence and does not look good to me after the rendering i.e. https://github.com/thatdocslady/minishift-addons/blob/bacb01fc3697f856bb8a59fe011ac60f0cd9f15b/README.adoc

I would suggest we just mention `Welcome`.